config.ini

```diff
@@ -32,7 +32,7 @@ notebooks = *.md  */*.md
 resources = img/ d2l/ d2l.bib setup.py
 
 # Files that will be skipped.
-exclusions = README.md STYLE_GUIDE.md INFO.md CODE_OF_CONDUCT.md CONTRIBUTING.md contrib/*md **/*_en.md */*_origin.md
```

Comment:
@semercim why do we remove `_en.md` file exclusions? In the future, if the translation repos need the `en` English markdown files in the git repo, then this removal will also make them build. For now i guess these are origin files which are also not part of the repo though.

If there is a good enough reason to remove it, maybe we can get rid of this then. Otherwise let's keep this unchanged.

Reply:
@AnirudhDagar , _en files were archaic. When I first started to translate the files, I named the original files with _en instead of _origin postfix.  With the latest update, there is no more _en files, all the original files have _origin postfix. Therefore, there is no need to keep _en files in the exclusion file.